paths/figma_attachment_keys/create.yaml

```diff
@@ -0,0 +1,44 @@
+---
+summary: Attach the Figma attachment to a key
+description: Attach the Figma attachment to a key
+operationId: figma_attachment_key/create
```

Comment:
wouldn't it be more logical to add these operations to `figma_attachments` (or perhaps `keys`) instead?

Reply:
Adjusted, not exactly sure if this is what you meant 
[CR](https://github.com/phrase/openapi/pull/415/commits/f4872f5e44bec82004dfae7f99b2d3260737a6fb)

Reply:
yeah, but they'd also have to live on `paths/figma_attachments` IMO

Reply:
but I guess the API PR has already merged in phrase?

Reply:
Yes the API PR is merged. Can we have two `create.yaml`/`destroy.yaml` in the same folder? I could adjust this. On the API PR we agreed to not use custom named methods and instead went with second controller to have option for second create/destroy paths